roles/openshift_storage_glusterfs/defaults/main.yml

```diff
@@ -14,6 +14,8 @@ openshift_storage_glusterfs_block_version: 'latest'
 openshift_storage_glusterfs_block_host_vol_create: True
 openshift_storage_glusterfs_block_host_vol_size: 100
 openshift_storage_glusterfs_block_host_vol_max: 15
+openshift_storage_glusterfs_block_storageclass: True
```

Comment:
This should default to `False` for now. We currently do not support glusterblock for anything other than Logging and Metrics, so a generally-available SC is no something we should provide yet.

Reply:
Aye aye, captain :)